### DISCUSSION TITLE
Images do not always show in upload filed type

### DISCUSSION THREAD

**Discussion by TomSmedley** [payloadcms/payload#895] [Feature Requests & Ideas]
https://www.loom.com/share/0ea34b50a15f40a1bf27acb8f0a76bfe

I'm not doing an official bug report for this yet as I would like to do further investigation, but just to start the dialog. Sometimes when uploading an image the preview will not happen. If this happens the field will become frozen.

**Comment by khakimvinh** [comment]
Hi, can you look into the browser console or the terminal output when the issue happen? That should give much more insight.
